app/script/localization/Localizer.js

```diff
@@ -70,6 +78,53 @@ class Localizer {
 
 z.localization.Localizer = new Localizer();
 
+z.l10n = (() => {
+
+  function replaceWithString(string, substitute) {
+    return string.replace(/{{\w+}}/, substitute);
+  }
+
+  function replaceWithObject(string, substitute) {
+    for (let identifier in substitute) {
+      if (substitute.hasOwnProperty(identifier)) {
+        string = string.replace(`{{${identifier}}}`, substitute[identifier]);
```

Comment:
This will only change the first occurrence, so it will fail on this one:

```js
replaceWithObject('{foo} {bar} {foo}', {'foo': 'FOO', 'bar': 'BAR'})
```

Maybe something like [this](http://stackoverflow.com/a/17606289/8418):

```js
string = string.replace(new RegExp(`{{${identifier}}}`, 'g'), substitute[identifier]);
```